### ISSUE TITLE
Upgrade from 0.95.0 to 0.98.0 or 0.99.0 fails

### ISSUE THREAD

**Issue by decibel** [theory/pgtap#213]
Except from [this build](https://travis-ci.org/decibel/pgtap/jobs/592824392#L838):

```
! ERROR:  column c.relhaspkey does not exist
! LINE 7:              WHERE c.relhaspkey = true
```

This is similar to #201; something in the catalog changed and we're not handling it correctly in our upgrade script. This is how [the upgrade script was fixed](https://github.com/decibel/pgtap/blob/fix_tap_funky/compat/10/pgtap--0.98.0--0.99.0.patch) for #201. In particular, note that the upgrade script fix is happening in a much later version (yet to be stamped, but presumably will be 1.1).

**Comment by theory** [comment]
Ugh, well I guess apply the same patch to the other upgrade scripts that mention `c.relhaspkey`.

**Comment by nasbyj** [comment]
This is fixed in 1.1.0
